src/directives/ctr-input.ts

```diff
@@ -157,6 +158,19 @@ export class CtrInput {
         }
     }
 
+    @HostListener("click", ["$event"])
+    public onClick(event: any) {
+        this.el.nativeElement.select();
+
+        if (this.openOnClick) {
+            if (this.completer.isOpen) {
+                this.completer.clear();
```

Comment:
When the dropdown is opened and we click again, we need to use `clear` to close it.